docs/xchain.js

```diff
@@ -1033,3 +1040,36 @@ function makeRawRequestInit(stringBody) {
     body: stringBody,
   };
 }
+
+function fillNetworkIdFromQueryString() {
+  const requestKey = getParameterByName('network');
+  if (requestKey) {
+    document.getElementById('pact-id').value = requestKey;
```

Comment:
I think this should be `document.getElementById('networkId')`

Reply:
This is the `requestKey`. The element for that has id `pact-id`

Reply:
I think the `const` is named wrong, since you're retrieving the `network` parameter 😁